_posts/0000-01-03-dingonix.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is a caption.
 
 ![cooler-octocat](https://octodex.github.com/images/twenty-percent-cooler-octocat.png)
```

Comment:
Hey, maybe use a different image from the octodex? https://octodex.github.com

Reply:
good idea! 😎